cabby/geo/walk.py

```diff
@@ -432,36 +463,27 @@ def get_number_intersections_past(main_pivot: GeoSeries, route: GeoDataFrame,
     The number of intersections between the main_pivot and goal. 
     If the main_pivot and goal are on different streets return -1.
   '''
-  dist = route['geometry'].apply(
-    lambda x: x.distance(main_pivot['geometry']))
-  pivot_idx = dist.idxmin().tolist()
-  pivot_goal_route = route.loc[pivot_idx:]
-
-  if pivot_goal_route.shape[0] <= 1:
-    return -1
 
-  # Check if main pivot is in the pivot_goal_route segment.
-  segment_pivot = LineString(pivot_goal_route['geometry'].iloc[0:2].values)
-  number_intersection = -util.project_point_in_segment(
-    segment_pivot, main_pivot['geometry'])
+  pivot_goal_route = compute_route_from_nodes(main_pivot['osmid'], 
```

Comment:
nit: Bring the first arg down a line, indent 4 spaces, and bring the other args back to 4 space indent as well

Reply:
changed